README.md

```diff
@@ -1,3 +1,5 @@
+# Checkstyle
```

Comment:
I don't like it:

![image](https://user-images.githubusercontent.com/900805/97673759-5c8f1280-1a9d-11eb-923b-4e85043fc721.png)

Now we have double title - one in the text, another in the image

Reply:
Can you do something like:
````
# ![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
````

Reply:
It works:

![image](https://user-images.githubusercontent.com/900805/97704092-cff74b00-1ac2-11eb-9cab-fed010566358.png)

much better now